packages/app-mobile/components/screens/status.tsx

```diff
@@ -1,50 +1,68 @@
-const React = require('react');
-
-const { View, Text, Button, FlatList } = require('react-native');
-const Setting = require('@joplin/lib/models/Setting').default;
-const { connect } = require('react-redux');
-const { ScreenHeader } = require('../ScreenHeader');
-const ReportService = require('@joplin/lib/services/ReportService').default;
-const { _ } = require('@joplin/lib/locale');
-const { BaseScreenComponent } = require('../base-screen');
-const { themeStyle } = require('../global-style');
-
-class StatusScreenComponent extends BaseScreenComponent {
-	static navigationOptions() {
+import * as React from 'react';
+
+import { View, Text, Button, FlatList, TextStyle, StyleSheet } from 'react-native';
+import Setting from '@joplin/lib/models/Setting';
+import { connect } from 'react-redux';
+import { ScreenHeader } from '../ScreenHeader';
+import ReportService, { ReportSection } from '@joplin/lib/services/ReportService';
+import { _ } from '@joplin/lib/locale';
+import { BaseScreenComponent } from '../base-screen';
+import { themeStyle } from '../global-style';
+import { AppState } from '../../utils/types';
+import checkDisabledSyncItemsNotification from '@joplin/lib/services/synchronizer/utils/checkDisabledSyncItemsNotification';
+import { Dispatch } from 'redux';
+
+interface Props {
+	themeId: number;
+	dispatch: Dispatch;
+}
+
+interface State {
+	report: ReportSection[];
+}
+
+class StatusScreenComponent extends BaseScreenComponent<Props, State> {
+	public static navigationOptions(): any {
```

Comment:
Based on the name, this seems to be related to [React Navigation](https://reactnavigation.org/docs/1.x/navigation-options/).

Reply:
Yes probably. That was added a long time ago and never removed since I wasn't sure if it did something or not.